clinica/converters/genfi_to_bids/_utils.py

```diff
@@ -859,7 +858,7 @@ def write_bids(
                 / str(metadata.session_id)
                 / f"{metadata.participant_id}_{metadata.session_id}_scan.tsv"
             )
-            metadata.source_path = metadata.source_path.relative_to(source)
+            metadata.source_path = metadata.source_path.relative_to(source).parent
```

Comment:
I'm not sure to understand. This isn't really fixing the non deterministic issue but avoiding it by writing the parent folder instead, right ?
I'm afraid we will lose the information about which dicom was used like this.